Makefile

```diff
@@ -131,11 +140,36 @@ lint-static-check:
 lint: lint-static-check
 	$(LINT) run --timeout 5m
 
+.PHONY: multimod-verify
+multimod-verify: install-tools
+	@echo "Validating versions.yaml"
+	multimod verify
```

Comment:
```suggestion
multimod-verify: | $(MULTIMOD)
	@echo "Validating versions.yaml"
	$(MULTIMOD) verify
```

Consistency.

Reply:
You are right. Another typo from me. Changed.